src/Collections/Parameters.php

```diff
@@ -50,12 +50,13 @@ protected function doExport()
     /**
      * Searches whether a parameter with the given unique combination exists.
      */
-    public function has(string $name, string $in): bool
+    public function has(string $name, string $in = null): bool
     {
-        return isset($this->parameters[$name.'/'.$in]);
+        $id = $in ? $name.'/'.$in : $name;
+        return isset($this->parameters[$id]);
     }
 
-    public function get(string $name, string $in): Parameter
+    public function get(string $name, string $in = null): Parameter
```

Comment:
This looks wrong to me, I would revert. I don't see why we would ever search a parameter by the reference it uses.